disco/types/permissions.py

```diff
@@ -11,7 +11,7 @@ class Permissions(object):
     ADD_REACTIONS = 1 << 6
     VIEW_AUDIT_LOG = 1 << 7
     PRIORITY_SPEAKER = 1 << 8
-    READ_MESSAGES = 1 << 10
+    VIEW_CHANNEL = 1 << 10
```

Comment:
Won't this break the API for existing code?

Reply:
If you're using Staging then it'll break your bot for permission calls sure. But that's the only call for it in this repo.

When sending things to Discord's API, it sends it as a number anyways.